techniques/system/common/1.0/rudder_stdlib_core.st

```diff
@@ -139,7 +139,7 @@ bundle agent rudder_common_report_index(technique_name, status, identifier, comp
         action    => immediate;
 
   reports:
-    full_compliance|send_reports::
+    full_compliance|reports_disabled|send_reports::
```

Comment:
I think so, since you only have the outputs  You are not able to understand what was done

At least in changes-only you can look into rudder compliance reporting